### PR TITLE
AzureStack Commerce Admin replacement.

### PR DESCRIPTION
**Pull request by deathly809**
Updated Commerce to be correct.  The old spec was incorrect in many ways and was not using the Admin namespace as it should.  

### PR information
- [ X] The title of the PR is clear and informative.
- [ X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [] If applicable, the PR references the bug/issue that it fixes.
- [X ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/azsadmin/resource-manager/commerce/readme.md [hunk 1]
```diff
@@ -33,24 +33,12 @@ These settings apply only when `--tag=package-2015-06-01-preview` is specified o
...
 input-file:
-    - Microsoft.Commerce.Admin/preview/2015-06-01-preview/CommerceAdmin.json
+    - Microsoft.Commerce.Admin/preview/2015-06-01-preview/Commerce.json
 ```
 
 ---
 # Code Generation
 
-
-## Swagger to SDK
```
Comment (jhendrixMSFT) [inline specification/azsadmin/resource-manager/commerce/readme.md]: Why was this removed?  Comment applies to everything else below too.

**Review by bganapa — APPROVED** [review]

File: specification/azsadmin/resource-manager/commerce/readme.md (hunk 1 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/commerce/readme.md]: Not sure, added it back.

**Comment by jhendrixMSFT** [comment]
@deathly809 There's still missing content in the README, presumably it all needs to be added back yes?

**Comment by deathly809** [comment]
@jhendrixMSFT We are only going to support C# for now.  Should I remove all references to other languages?

**Comment by jhendrixMSFT** [comment]
@deathly809 I see, if this is the case then yes please remove the other languages and be sure to add a note in the README that only C# is supported at present.  Do keep in mind though that having a swagger in the public repo means it's available for general consumption so there's nothing stopping folks from generating their own SDK in the language of their choice.

**Comment by deathly809** [comment]
@jhendrixMSFT Done.  Let me know if I need to do anything else, thanks.

**Comment by jhendrixMSFT** [comment]
@deathly809 I just realized we've already published this package for Go.  Does this mean we should remove it?  I don't know about Java, @RikkiGibson can you comment?

**Comment by deathly809** [comment]
@jhendrixMSFT Yeah that should be deleted.

**Comment by RikkiGibson** [comment]
Looks like we didn't publish a library on the Java side.

**Review by jhendrixMSFT — APPROVED** [review]
